Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -294,12 +294,14 @@ static u32 AdvanceDTK(u32 maximum_samples, u32* samples_to_process)
 static void DTKStreamingCallback(DIInterruptType interrupt_type, const std::vector<u8>& audio_data,
                                  s64 cycles_late)
 {
-  // TODO: Should we use GetAISSampleRate instead of a fixed 48 KHz? The audio mixer is using
-  // GetAISSampleRate. (This doesn't affect any actual games, since they all set it to 48 KHz.)
-  const u32 sample_rate = AudioInterface::Get48KHzSampleRate();
+  // Actual games always set this for 48 KHz
+  // although if it were changed to 32KHz, it would likely wreak havoc on audio code
```

Comment:
This line of this comment is a little confusing. I'm assuming that what you mean is that if we use `AudioInterface::Get48KHzSampleRate`, there would be havoc if the game changed to 32 KHz, but once this is merged and this comment is read without the context of the changeset, the connection to the now removed call to `AudioInterface::Get48KHzSampleRate` would be unclear. Maybe change the comment to something like "Actual games always set this to 48 KHz, but let's make sure to use GetAISSampleRateDivisor() just in case it changes to 32 KHz". (Feel free to reword)